### DISCUSSION TITLE
what's the right command to  recreate  yarn.lock after it's creation?

### DISCUSSION THREAD

**Discussion by viraj124** [PaulRBerg/hardhat-template#88] [Q&A] [answered]
when I run just yarn install I get this error
```
Usage Error: The project in /Users/viraj/Development/package.json doesn't seem to have been installed - running an install there might help

$ yarn install [--json] [--immutable] [--immutable-cache] [--check-cache] [--inline-builds] [--mode #0]
```
tried diff. command combinations non e of them worked wondering what is the right way

**Comment by viraj124** [comment]
@paulrberg can you check this when you get time thanks

**Comment by SvenMeyer** [comment]
you can try fixing it by creating an (empty) `yarn.lock` file.
```
$ touch yarn.lock
```

**Comment by PaulRBerg** [comment, marked as answer]
@viraj124 This doesn't seem to be a question about my Solidity template. If you want to use this, clone it in a separate repo, and run `yarn install`. It will work.

Whatever you have in the `Development` folder is not a node.js project.

**Comment by viraj124** [comment]
yeah sorry about that there was a package.json which i didn't notice my bad, but if possible can I ask a hardhat related query here ?

I am trying to fork polygon but on even calling a view method in the external contract I get
`transaction reverted without a reason`
I have tried to increase gas but that didn't really have any effect

**Reply by PaulRBerg** [reply]
Unfortunately, this Q&A channel is not dedicated to generic questions about either Ethereum or Hardhat. I recommend you post your question in the [Hardhat Discord server](https://twitter.com/HardhatHQ). Pro tip: make sure to add plenty of details..
